src/views/NoTrialsFound/NoTrialsFound.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { useTracking } from 'react-tracking';
+
+import { NoResults } from '../../components';
+import { useStateValue } from '../../store/store';
+import { TokenParser } from '../../utils';
+import { CISBanner } from '../../components';
```

Comment:
This can be added to the import on line 6, since they're from the same place.

Reply:
Fixed!